libraries/Adapters/Microsoft.Bot.Builder.Adapters.Webex/WebexClientWrapper.cs

```diff
@@ -23,6 +25,47 @@ public class WebexClientWrapper
         private const string ActionsUrl = "https://api.ciscospark.com/v1/attachment/actions";
 
         private TeamsAPIClient _api;
+        private readonly WebexAdapterOptions _config;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="WebexClientWrapper"/> class.
+        /// Creates a Webex Client Wrapper. See <see cref="WebexAdapterOptions"/> for a full definition of the allowed parameters.
+        /// </summary>
+        /// <param name="config">An object containing API credentials, a webhook verification token and other options.</param>
+        /// <param name="webexClient">A Webex API interface.</param>
+        public WebexClientWrapper(WebexAdapterOptions config)
+        {
+            _config = config ?? throw new ArgumentNullException(nameof(config));
+
+            if (string.IsNullOrWhiteSpace(_config.AccessToken))
+            {
+                throw new Exception("AccessToken required to create controller");
```

Comment:
I think that we should use ArgumentException instead of Exception ([link](https://stackoverflow.com/questions/8459755/argumentexception-vs-argumentnullexception))

Reply:
Feedback applied. Thanks!